doc/source/advanced-use.rst

```diff
@@ -6,246 +6,256 @@ Clones
+type, as this process only works with basejails. Running:
 
-Verify whether the jail is a basejail:
+:samp:`# iocage get type UUID|TAG`
 
-``iocage get type UUID|TAG``
+needs to return "basejail", for the desired jail.
 
-Should return type "basejail".
+Upgrading can be forced while the jail is online by executing:
 
-The upgrade can be forced while the jail is online with executing:
+:samp:`# iocage upgrade UUID|TAG`
 
-  ``iocage upgrade UUID|TAG``
+This forcibly re-clones the basejail filesystems while the jail is
+running (no downtime) and update the jail's :file:`/etc` with the
+changes from the new RELEASE.
 
-This will forcibly re-clone the basejail filesystems while the jail is running (no downtime) and update the jails /etc with the changes from the new RELEASE.
+To upgrade the jail while it is stopped, run:
 
-``iocage set release=10.1-RELEASE UUID|TAG``
+:samp:`# iocage set release=11.0-RELEASE UUID|TAG`
 
-This will cause the jail to re-clone its filesystems from 10.1-RELEASE on next jail start. This will not update the jails /etc files with changes from the next RELEASE.
+This causes the jail to re-clone its filesystems from the 11.0-RELEASE
+on next jail start. This does not update the jail's :file:`/etc` files
+with changes from the next RELEASE.
 
 Auto boot
 ---------
 
-Make sure ``iocage_enable="YES"`` is set in ``/etc/rc.conf``.
+Make sure :command:`iocage_enable="YES"` is set in :file:`/etc/rc.conf`.
 
-To enable a jail to auto-boot during a boot, simply run:
+To enable a jail to auto-boot during a system boot, simply run:
 
-``iocage set boot=on UUID|TAG``
+:samp:`# iocage set boot=on UUID|TAG`
 
 Boot priority
 +++++++++++++
 
 Boot order can be specified by setting the priority value:
 
-``iocage set priority=20 UUID|TAG``
+:samp:`# iocage set priority=20 UUID|TAG`
 
-Lower value means higher boot priority.
+*Lower* values are higher in the boot priority.
 
 Snapshot management
 -------------------
 
 iocage supports transparent ZFS snapshot management out of the box.
-Snapshots are point-in-time copies of data, a safety point to which a jail can be reverted at any time.
-Initially snapshots take up almost no space as only changing data is recorded.
+Snapshots are point-in-time copies of data, a safety point to which a
+jail can be reverted at any time. Initially, snapshots take up almost no
+space, as only changing data is recorded.
 
 List snapshots for a jail with:
 
-``iocage snaplist UUID|TAG``
+:samp:`# iocage snaplist UUID|TAG`
 
-To create a new snapshot run:
+To create a new snapshot, run:
 
-``iocage snapshot UUID|TAG``
+:samp:`# iocage snapshot UUID|TAG`
 
-This will create a snapshot based on current time.
+This creates a snapshot based on current time.
 
-If you'd like to create a snapshot with custom naming run:
+To create a snapshot with custom naming, run:
 
-``iocage snapshot UUID|TAG@mysnapshotname``
+:samp:`# iocage snapshot UUID|TAG@mysnapshotname`
 
 Resource limits
 ---------------
```

Comment:
Disclaimer that this is only for legacy implementations, it has not been written for the new version as of yet.